romana-install/roles/romana/install-agent/templates/config/romana.yaml

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This looks very much like JSON to me. Why is the extension for this file ".yaml"?